### DISCUSSION TITLE
Does the walletconnect v2.0 support venly and bitski wallets as well ?

### DISCUSSION THREAD

**Discussion by 0xpolydexter** [WalletConnect/walletconnect-monorepo#1997] [Developer Support]
Does the walletconnect v2.0 support venly and bitski wallets as well ?

**Comment by karelstriegel** [comment]
@hiteshgawhade, We are actively working on getting Venly ready for v2 of wallet connect, our internal deadline is end of Febr.

**Reply by KayCm** [reply]
how to download ? can`t found in google play & app store

**Reply by finessevanes** [reply]
https://get.venly.io/business
